Model/Indexer/AbstractIndexer.php

```diff
@@ -136,25 +135,22 @@ public function __construct(
     abstract public function getIndexerId(): string;
 
     /**
-     * @inheritdoc
+     * @inheritDoc
      * @throws Exception
      */
     public function executeFull()
     {
         if ($this->allowFullExecution() === true) {
-            $indexerId = $this->getIndexerId();
-            $message = sprintf(
-                'Begin a full reindex for indexer "%s"',
-                $indexerId
-            );
-            $this->nostoLogger->info($message);
+            $this->logInfo('Begin a full reindex');
             $this->doWork();
-            $this->nostoLogger->info('Finished full reindex');
+            $this->logInfo('Finished full reindex');
+        } else {
+            $this->logInfo('Full reindex is disabled in Nosto module settings');
```

Comment:
This is a bit misleading to be logged when you call `setup:upgrade`. Maybe something like this would be clearer:
```suggestion
            $this->logInfo('Full reindex is either disabled in Nosto module settings or called from setup:upgrade');
```